src/main/java/me/gnat008/perworldinventory/data/FlatFile.java

```diff
@@ -41,21 +24,19 @@
 import java.io.IOException;
 import java.util.UUID;
 
-public class FileWriter implements DataWriter {
+public class FlatFile implements DataSource {
 
-    private final File FILE_PATH;
+    private final File DATA_FOLDER;
 
-    private final PerWorldInventory plugin;
-    private final PlayerSerializer playerSerializer;
-    private final PWIPlayerFactory pwiPlayerFactory;
+    private PerWorldInventory plugin;
+    private PlayerSerializer playerSerializer;
+    private PWIPlayerFactory pwiPlayerFactory;
```

Comment:
And then I would have the fields or constructor in FlatFile have the `@Inject` annotation? (With the second option)